### PR TITLE
week08-Ati Takrami

### PR DESCRIPTION
**Pull request by takrami**
Live link : https://focused-snyder-bb829c.netlify.app/

I created my own repository by mistake instead of forking it from technigo's. I submitted the link from my own repository instead on canvas.
Here is the link from my repo: [My Repository](https://github.com/takrami/portfolio)
> Please consider that I copied the whole code from my repo. This is the reason this PR contains 2 commits.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+              Simple currency exchange app using on React, CSS-Grid, i18n,
+              styled-component
+            </p>
+            <div class="projects-item-tech">
+              <span class="projects-item-tech-tags">React</span>
+              <span class="projects-item-tech-tags">CSS-Grid</span>
+              <span class="projects-item-tech-tags">i18n</span>
+              <span class="projects-item-tech-tags">styled-component</span>
+            </div>
+          </li>
```
Comment (KarolCislowski) [inline src/index.html]: Your project needs to be a link to the netlify hosted version of it.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+              class="projects-item-img res-img"
+            />
+            <time class="time">DES 2019</time>
+            <h3 class="">Currency exchange</h3>
+
+            <p class="projects-item-description">
+              Simple currency exchange app using on React, CSS-Grid, i18n,
+              styled-component
+            </p>
+          </li>
```
Comment (KarolCislowski) [inline src/index.html:287]: Your thoughts should be a link to a full version of your blog post.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Great job! You just need to convert your project/post cards to the links.

File: src/index.html (hunk 1 above)
Comment (takrami) [reply src/index.html]: This is the live version: https://focused-snyder-bb829c.netlify.app/

**Comment by takrami** [comment]
Thank you so much for your comments. I added the links to my projects.

**Review by KarolCislowski — APPROVED** [review]
nice work 👍🏻

**Pull request closed without merging**